src/simplewallet/simplewallet.cpp

```diff
@@ -6379,7 +6388,27 @@ static bool parse_lns_name_string(std::vector<std::string> const &args, size_t f
   return true;
 }
 //----------------------------------------------------------------------------------------------------
-bool simple_wallet::buy_lns_mapping(const std::vector<std::string>& args)
+std::string eat_named_argument(std::vector<std::string> &args, char const *prefix, size_t prefix_len)
```

Comment:
The 2nd & 3rd arguments here would be nicely replaced with a string_view.

Reply:
String viewify-pr, either before if I can fit it in, otherwise after